library_generation/DEVELOPMENT.md

```diff
@@ -48,32 +48,36 @@ Although the scripts are designed to be run in a Docker container, you can also
 run them directly. This section explains how to run the entrypoint script
 (`library_generation/cli/entry_point.py`).
 
-## Installing prerequisites
+## Assumptions made by the scripts
+### The Hermetic Build's well-known folder
+Located in `${HOME}/.library_generation`, this folder is assumed by the scripts
+to contain the generator JAR. Please note that this is a recent feature and only
+this jar is expected to be there. Soon enough, more binaries such as the gRPC
+and protoc plugins will also be stored here. Developers must make sure
+this folder is properly configured before running the scripts locally.
+Note that this relies on the `HOME` en var which is always
```

Comment:
nit: one sentence per line.

Reply:
I found [this article](https://asciidoctor.org/docs/asciidoc-recommended-practices/#one-sentence-per-line) that confirms this is a good practice. I reformatted the file.